drivers/clock_control/nrf5_power_clock.c

```diff
@@ -15,6 +15,7 @@
 #include <nrf_power.h>
 #include <drivers/clock_control/nrf5_clock_control.h>
 #endif
+#include "nrf_clock.h"
```

Comment:
Minor, could you move this above the #if clause, just to keep non-conditional includes together unless dependency requirements.

Reply:
done